espnet2/asr/decoder/linear_decoder.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         self,
         vocab_size: int,
         encoder_output_size: int,
-        pooling: str = "CLS",
+        pooling: str = "mean",
```

Comment:
is this change OK?
Woudn't it affect the others?

Reply:
This is okay because the only recipe which uses it already defines the pooling in config: https://github.com/espnet/espnet/blob/master/egs2/esc50/asr1/conf/beats_classification.yaml#L75

I think mean is a better default than CLS therefore I changed it.

Reply:
Is this used for the other recipes?
I just want to make sure that it does not affect other people's work

Reply:
No, I introduced this decoder in https://github.com/espnet/espnet/pull/5977
Nobody since then has touched it: https://github.com/search?q=repo%3Aespnet%2Fespnet%20linear_decoder&type=code

Reply:
Thanks for the clarification!